pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -152,6 +154,7 @@
   withVulkan ? withHeadlessDeps && !stdenv.hostPlatform.isDarwin,
   withVvenc ? withFullDeps && lib.versionAtLeast version "7.1", # H.266/VVC encoding
   withWebp ? withHeadlessDeps, # WebP encoder
+  withWhisper ? withFullDeps && !stdenv.hostPlatform.isDarwin && lib.versionAtLeast version "8.0",
```

Builds fine on darwin.
```suggestion
  withWhisper ? withFullDeps && lib.versionAtLeast version "8.0",
```